src/arch/arch_default.h

```diff
@@ -38,6 +43,12 @@
 #else
 #define DEFAULT_INPUT_DRIVER_LIST "X11"
 #endif
+
+#if defined(WITH_SDL3)
+#undef DEFAULT_INPUT_DRIVER_LIST
+#define DEFAULT_INPUT_DRIVER_LIST "sdl"
```

Comment:
SDL3's lowlevelwindow requires the associated input handler.

Reply:
Given that your `arch_default.h` is made up of macros, it seems like you're working based off the `release` branch.  In `beta`, `arch_default.h` was redone to use vectors of strings in `beta`, so please be sure to work off of that branch.